massa-grpc/src/public.rs

```diff
@@ -15,7 +15,7 @@ use massa_models::address::Address;
 use massa_models::amount::Amount;
 use massa_models::block::{Block, BlockGraphStatus};
 use massa_models::block_id::BlockId;
-use massa_models::config::CompactConfig;
+use massa_models::config::{CompactConfig, CHAINID};
```

Comment:
here as well, pass it through the config

Reply:
Fixed